numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -256,3 +258,66 @@ def get_debian_pkg_libdevice():
     if not os.path.exists(pkg_libdevice_location):
         return None
     return pkg_libdevice_location
+
+
+def get_current_cuda_target_name():
+    """Determine conda's CTK target folder based on system and machine arch.
+
+    CTK's conda package delivers headers based on its architecture type. For example,
+    `x86_64` machine places header under `$CONDA_PREFIX/targets/x86_64-linux`, and
+    `aarch64` places under `$CONDA_PREFIX/targets/sbsa-linux`. Read more about the
+    nuances at cudart's conda feedstock:
+    https://github.com/conda-forge/cuda-cudart-feedstock/blob/main/recipe/meta.yaml#L8-L11  # noqa: E501
+    """
+    system = platform.system()
+    machine = platform.machine()
+
+    if system == "Linux":
+        arch_to_targets = {
+            'x86_64': 'x86_64-linux',
+            'aarch64': 'sbsa-linux'
+        }
+        return arch_to_targets.get(machine)
+
+    return None
+
+
+def get_conda_include_dir():
+    """
+    Return the include directory in the current conda environment, if one
+    is active and it exists.
+    """
+    conda_prefix = os.environ.get('CONDA_PREFIX')
```

Comment:
It's not clear to me that Python is always executed with the conda environment active. The other functions in this file, like `get_conda_ctk()`, check for the presence of `conda-meta` under `sys.prefix` to determine if it's running in a conda environment (and to more robustly defend against `CONDA_PREFIX` being something other than where the current Python interpreter is, I suppose). Is there a strong reason for using `CONDA_PREFIX` instead? If not, can we be consistent with the method used for `get_conda_ctk()`, so that we have less chance of getting two different results for the libraries and the includes?

Reply:
> Is there a strong reason for using CONDA_PREFIX instead?

Not really, I didn't know of the `sys.prefix` way to get conda paths.

Reply:
Also fixed in https://github.com/NVIDIA/numba-cuda/pull/17/commits/9b6dda9dd16ec678c303c9fcb733de80e5a4c628